app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        OutdatedVaccineError,
+                        NotVaccinatedError)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The return message for `OutdatedVaccineError` is the same as for `NotVaccinatedError`. Consider providing a distinct message, such as 'All friends should have up-to-date vaccinations', to differentiate between the two cases.